### ISSUE TITLE
[storybook] Enable A11Y addon

### ISSUE THREAD

**Issue by CubLion** [ing-bank/lion#298]
One of the selling points of Lion is the fact that it is Accessible and yet Storybooks is missing the [A11Y addon](https://github.com/storybookjs/storybook/tree/master/addons/a11y) that would make this feature even more visible.

**Comment by NioniosSpeed** [comment]
Hey, is it ok to pick this up? :-)

**Comment by gerjanvangeest** [comment]
Yes please do!

**Comment by r4zorgeek** [comment]
@gerjanvangeest Can I take this?

**Comment by gerjanvangeest** [comment]
I don't mind who picks it up. @NioniosSpeed also asked to add this. But there are a lot of demos. See if you guys can work on it together or split it up? Say `form components` and `non-form components`?
